pkg/specgenutil/volumes.go

```diff
@@ -22,6 +22,12 @@ var (
 	errNoDest    = errors.New("must set volume destination")
 )
 
+type universalMount struct {
+	mount   spec.Mount
```

Comment:
if we embed the struct we would not need to add the extra .mount everywhere

Reply:
I'd prefer it that way if we could just cast the universalMount to a spec.Mount after, but we can't. So it saves us a few changes in parseMountOptions but hurts more than helps in the callers.

Reply:
ok fair enough